src/utils/bip39.ts

```diff
@@ -1,2 +1,18 @@
+// Copyright 2018 - 2021 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
 export const bip39Words =
```

Comment:
Is it a good practice to store such a long string in memory ?

Reply:
The string seems to be ~13KB
```
> new Blob([bip39words]).size
13115
```

or in Node:

```
> Buffer.byteLength(bip39words).length
13115
```

Not sure what other approach there is in order to pass this list of words into the `Tags` component of the `@yaireo/tagify/dist/react.tagify` library so that when importing a wallet, a list of suggested words pop up when the user types their words.